src/ok.rs

```diff
@@ -152,6 +157,21 @@ pub fn cmd(gen: Option<SevGeneration>, quiet: bool) -> Result<()> {
         }
     }
 
+    // Complete the rest of the system tests.
+    if passed {
+        let read_test = &mut sysfs_tests[0](fs::OpenOptions::new().read(true));
+        read_test.msg = format!("Reading from /dev/sev: {}", read_test.msg);
+        if !check_test(read_test, quiet) {
+            passed = false;
+        }
+
+        let write_test = &mut sysfs_tests[1](fs::OpenOptions::new().write(true));
+        write_test.msg = format!("Writing to /dev/sev: {}", write_test.msg);
+        if !check_test(write_test, quiet) {
+            passed = false;
+        }
```

Comment:
If we change the order on the tests, we have to remember to update these. It'd be better if we could just iterate through these tests and not require knowledge of the test ordering for interpreting the results. I'll add a separate comment where the Vec is made.